presto-main/src/main/java/io/prestosql/connector/system/TransactionsSystemTable.java

```diff
@@ -62,7 +65,7 @@ public TransactionsSystemTable(Metadata metadata, TransactionManager transaction
                 .column("isolation_level", createUnboundedVarcharType())
                 .column("read_only", BOOLEAN)
                 .column("auto_commit_context", BOOLEAN)
-                .column("create_time", TIMESTAMP_MILLIS)
+                .column("create_time", TIMESTAMP_TZ_MILLIS)
```

Comment:
I am a bit concerned about backward compatibility of this change. Not sure how commonly are those system schemas used by Presto users.

Reply:
See 
https://github.com/prestosql/presto/pull/4772
https://github.com/prestosql/presto/pull/4756
https://github.com/prestosql/presto/pull/4753
https://github.com/prestosql/presto/pull/4752
i guess system connector simply got missed during that effort, but @electrum @dain may know better.

Reply:
fair